plugins/node/opentelemetry-instrumentation-hapi/package.json

```diff
@@ -60,8 +61,7 @@
   "dependencies": {
     "@opentelemetry/core": "^1.8.0",
     "@opentelemetry/instrumentation": "^0.51.0",
-    "@opentelemetry/semantic-conventions": "^1.0.0",
-    "@types/hapi__hapi": "20.0.13"
+    "@opentelemetry/semantic-conventions": "^1.0.0"
   },
   "homepage": "https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-hapi#readme"
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Odd. My editor is setup to use `.editorconfig`, but this wasn't auto-fixed.